src/main/react-webapp/src/components/Searchbar.js

```diff
@@ -0,0 +1,50 @@
+import {ReactSearchAutocomplete} from 'react-search-autocomplete';
+import {useHistory} from 'react-router-dom';
+import {useEffect, useState} from "react";
+
+const Searchbar = () => {
+
+ const [drugs, setDrugs] = useState([])
+
+ useEffect(() => {
+  const getData = async () => {
+	const data = await fetch('/list-drugs')
+	const allDrugs = await data.json()
+	setDrugs(allDrugs)
+  }
+  getData()
+ }, [])
+
+ let history = useHistory();
+
+ const handleOnSelect = (item) => {
+  // redirect the user to the selected site
+  history.push(`/drug/${item.id}`);
+ }
+
+ const formatResult = (item) => {
+  return item;
+ }
+
+ let items = [];
+ for(let drug of drugs){
+  items.push({id: drug.id, name: drug.name, aliases: drug.aliases});
```

Comment:
Is there any difference between `items` and the original `drugs` array? Why not just pass drugs as the value for ReactSearchAutocomplete.items?

Reply:
drugs.aliases was supposed to have a .join(' ') after it. Ill update it right now.